bundler/lib/bundler/friendly_errors.rb

```diff
@@ -100,15 +100,15 @@ def request_issue_report_for(e)
         #{issues_url(e)}
 
         If there aren't any reports for this error yet, please create copy and paste the report template above into a new issue. Don't forget to anonymize any private data! The new issue form is located at:
-        https://github.com/rubygems/bundler/issues/new
+        https://github.com/rubygems/rubygems/issues/new?labels=Bundler
```

Comment:
I didn’t put the [Bundler-related template](https://github.com/rubygems/rubygems/blob/master/.github/ISSUE_TEMPLATE/bundler-related-issue.md) here because the message already asks one to use the report template above.